libs/design-system/src/lib/Components/Footer/index.tsx

```diff
@@ -0,0 +1,170 @@
+import { Link } from 'react-router-dom';
+
+import { ReactComponent as FlowDocsLogo } from '../../Images/logos/flow-docs-logo-light.svg';
+import { ReactComponent as GithubIcon } from '../../Images/social/github-light.svg';
+import { ReactComponent as DiscordIcon } from '../../Images/social/discord-light.svg';
+import { ReactComponent as ForumIcon } from '../../Images/social/forum-light.svg';
+import { ReactComponent as OnFlowIcon } from '../../Images/logos/flow-icon-bw-light.svg';
+
+// reduce repetition of the section layout in Footer component
+const footerSections = [
+  {
+    header: 'Documentation',
+    links: [
+      {
+        link: '#', // TODO: replace with actual links
+        text: 'Getting Started',
+      },
+      {
+        link: '#',
+        text: "SDK's",
+      },
+      {
+        link: '#',
+        text: 'Tutorials',
+      },
+      {
+        link: '#',
+        text: 'Guides',
+      },
+      {
+        link: '#',
+        text: 'Concepts',
+      },
+      {
+        link: '#',
+        text: 'Walkthroughs',
+      },
+      {
+        link: '#',
+        text: 'How-To',
+      },
+      {
+        link: '#',
+        text: 'FAQ',
+      },
+      {
+        link: '#',
+        text: 'Tools',
+      },
+    ],
+  },
+  {
+    header: 'Community',
+    links: [
+      {
+        link: '#',
+        text: 'Ecosystem',
+      },
+      {
+        link: '#',
+        text: 'GitHub',
+      },
+      {
+        link: '#',
+        text: 'Discord',
+      },
+      {
+        link: '#',
+        text: 'Flowverse',
+      },
+      {
+        link: '#',
+        text: 'Forum',
+      },
+    ],
+  },
+  {
+    header: 'Start Building',
+    links: [
+      {
+        link: '#',
+        text: 'Flow Playground',
+      },
+      {
+        link: '#',
+        text: 'Crypto Kitties',
+      },
+    ],
+  },
+  {
+    header: 'Network',
+    links: [
+      {
+        link: '#',
+        text: 'Network Status',
+      },
+      {
+        link: '#',
+        text: 'Mainnet',
+      },
+      {
+        link: '#',
+        text: 'Testnet',
+      },
+      {
+        link: '#',
+        text: 'Canary',
+      },
+      {
+        link: '#',
+        text: 'Sporks FAQ',
+      },
+    ],
+  },
+];
+
+export function Footer() {
+  return (
+    <footer className="bg-black px-6 text-white">
+      <div className="container mx-auto">
+        <div className="block px-2 pt-8 pb-6 md:flex md:items-center md:justify-between md:px-4 md:pt-16">
+          <div>
+            <FlowDocsLogo className="h-10 w-min object-contain" />
+          </div>
+          <div className="flex items-center justify-start pt-8 lg:justify-center lg:pt-0">
```

Comment:
The `lg:pt-0` should be `md:pt-0` for better vertical alignment at medium widths.

Removing `px-3` on the links below, and adding a gap-6 on this div would also fix the icon left alignment on mobile